morgoth/base/static/js/actions/addons.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
```

Comment:
axios looks really similar to the almost-standard fetch API. Any reason not to just use fetch?